boards/sparky_v2.cpp

```diff
@@ -115,9 +115,9 @@ bool Sparky_v2::init(void)
     // -------------------------------------------------------------------------
     // Init LEDs
     // -------------------------------------------------------------------------
-    ret &= led_err_gpio_.init();
-    ret &= led_stat_gpio_.init();
-    ret &= led_rf_gpio_.init();
+    ret = led_err_gpio_.init();
+    ret = led_stat_gpio_.init();
+    ret = led_rf_gpio_.init();
```

Comment:
Yes indeed.
But anyway without debug message it is pointless to have two variables `ret` and `init_success`. The idea was to do:
```cpp
bool init_success = true;
bool ret;

ret = module_.init();
print_success("Module name", ret);
init_success &= ret;

[...]

print_success("BOARD", init_success);
```